include/kf/stl/new

```diff
@@ -2,31 +2,33 @@
 
 #pragma warning(push)
 #pragma warning(disable : 4595) // non-member operator new or delete functions may not be declared inline
+#pragma warning(disable : 4996) // TODO: ExAllocatePoolWithTag is deprecated, use ExAllocatePool2
 
-inline void* __cdecl operator new(size_t size, POOL_TYPE poolType)
+inline void* __cdecl operator new(size_t size, POOL_TYPE poolType) noexcept
 {
 #pragma warning(suppress: 28160) // Must succeed pool allocations are forbidden. Allocation failures cause a system crash.
-    return ::ExAllocatePoolWithTag(poolType, size ? size : 1, 'n++C');
+    return ::ExAllocatePoolWithTag(poolType, size == 0 ? 1 : size, 'n++C');
 }
 
-inline void __cdecl operator delete(void* ptr)
+inline void __cdecl operator delete(void* ptr) noexcept
 {
-    if (ptr)
-    {
-        ::ExFreePoolWithTag(ptr, 'n++C');
-    }
+    ::ExFreePoolWithTag(ptr, 'n++C');
```

Comment:
Oh, here is misunderstanding. `if (ptr)` is required for wrapping `ExFreePoolWithTag` and is not required for wrapping `delete`.

Reply:
++